### DISCUSSION TITLE
How do I know if a mention has been deleted?

### DISCUSSION THREAD

**Discussion by leo-petrucci** [ueberdosis/tiptap#2009] [General Questions]
The mention plugin is amazing! But I'm having some trouble using it.

So, every time a new mention is made, I add information about the user's account that's been mentioned to an array. This includes their `id`, as well as their `username`, it's all stuff I need to notify the user correctly in the backend.

The process works pretty well, but the problems come along when I need to delete one of those mentions: **How do I know when a specific mention has been removed from the editor so I can remove it from my array?**

Typescript tells me there's no onDelete callback or anything similar. Is there a workaround I'm missing?

**Comment by hanspagel** [comment]
What about storing the list of mentions in your backend and checking if one was removed there?

**Reply by leo-petrucci** [reply]
This is before the comment is actually sent to the backend. Are you suggesting sending the list of mentions, then parsing the editor content to check which mentions are still present?

That could work but I haven't found any way to append more data to the mention tags, so the only information I can get from them is the user's name. It would be useful if I could pass the user's id to the tag as well.

I've looked at the source code for both Suggestion and Mention but I'm not really clear on how it could be implemented.
